Makefile

```diff
@@ -28,6 +29,12 @@ helm-lint: helm-lint-$(notdir $(CHART_DIR)/*)
 helm-lint-%:
 	$(HELM) lint $(CHART_DIR)/$*
 
+deploy-cert-manager:
+	kubectl create namespace cert-manager
```

Comment:
helm will create the namespace
```suggestion
```

Reply:
Doesn't look like it. I got this error when I removed the create namespace step - 

```
docker run --rm -i --entrypoint="" --network host -e KUBECONFIG=/apps/.kube/config -v /Users/abalavenkata/.kube/:/apps/.kube/ -v /Users/abalavenkata/go/src/github.com/abalaven/konk:/apps -v /Users/abalavenkata/go/src/github.com/abalaven/konk/repositories.yaml:/tmp/.config/helm/repositories.yaml -v /Users/abalavenkata/go/src/github.com/abalaven/konk/jetstack-index.yaml:/tmp/.cache/helm/repository/jetstack-index.yaml infoblox/helm:3.2.4-5b243a2 helm upgrade -i --wait cert-manager --namespace cert-manager jetstack/cert-manager --version v1.0.1 \
		--set installCRDs=true \
		--set extraArgs[0]="--enable-certificate-owner-ref=true"
Release "cert-manager" does not exist. Installing it now.
Error: create: failed to create: namespaces "cert-manager" not found
make: *** [deploy-cert-manager] Error 1
```

Reply:
They changed the default behavior in helm3, you have to pass a flag to create the namespace https://helm.sh/docs/faq/#automatically-creating-namespaces

Reply:
Done, updated to include the `--create-namespace` flag.